packages/design-system/src/elements/toniq-input/toniq-input.element.ts

```diff
@@ -95,6 +95,8 @@ export const ToniqInput = defineToniqElement<{
     blockedInputs?: string | RegExp;
     /** Disable all browser helps like spellchecking, autocomplete, etc. */
     disableBrowserHelps?: boolean;
+    /** Any letters in the given string or matches to the given RegExp will be added as suffix */
+    suffix?: string | RegExp;
```

Comment:
Only a string should be allowed for this input:
```suggestion
    suffix?: string;
```